jupyterhub/tests/test_api.py

```diff
@@ -257,17 +233,13 @@ def test_add_multi_user(app):
         assert not user.admin
 
     # try to create the same users again
-    r = api_request(app, 'users', method='post',
-        data=json.dumps({'usernames': names}),
-    )
+    r = api_request(app, 'users', method='post', data=json.dumps({'usernames': names}), )
     assert r.status_code == 400
 
     names = ['a', 'b', 'ab']
 
     # try to create the same users again
-    r = api_request(app, 'users', method='post',
-        data=json.dumps({'usernames': names}),
-    )
+    r = api_request(app, 'users', method='post', data=json.dumps({'usernames': names}), )
```

Comment:
In general, I think trailing commas aren't used when unwrapping to a single line. I'd leave it as the wrapped lines, though.